content/accessibility/button-purpose.mdx

```diff
@@ -104,12 +102,13 @@ Each button in the example below has a unique name which describes its action (
   }
 </script>
 <ul>
-  <li>Apples <button onclick="removeItem(event)" aria-label="Remove 'Apples'" type="button">❌</button></li>
-  <li>Bananas <button onclick="removeItem(event)" aria-label="Remove 'Bananas'" type="button">❌</button></li>
-  <li>Cantaloupes <button onclick="removeItem(event)" aria-label="Remove 'Cantaloupes'" type="button">❌</button></li>
+  <li>Apples <button onclick="removeItem(event)" type="button">❌</button></li>
+  <li>Bananas <button onclick="removeItem(event)" type="button">❌</button></li>
+  <li>Cantaloupes <button onclick="removeItem(event)" type="button">❌</button></li>
```

Comment:
Ah okay I see – flipping the order to put the positive scenario first and the avoid scenario second.